From 57d68387d2a71d5288a00957f57fa3580490ce01 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Thu, 3 Jul 2025 14:33:01 +0200
Subject: [PATCH] media: stm32: csi: correct missing BGGRxx formats

Correct a typo leading to having twice RGGBxx formats and not
having BGGRxx.

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: Id7ab371643af25f60aaca6dfaab398b1241b8c69
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/472059
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
---
 drivers/media/platform/st/stm32/stm32-csi.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/media/platform/st/stm32/stm32-csi.c
+++ b/drivers/media/platform/st/stm32/stm32-csi.c
@@ -231,22 +231,22 @@ static const struct stm32_csi_fmts stm32
 	FMT_MBUS_DT_DTFMT_BPP(SGRBG8_1X8, RAW8, BPP8, 8),
 	FMT_MBUS_DT_DTFMT_BPP(SRGGB8_1X8, RAW8, BPP8, 8),
 	/* 10 bit */
-	FMT_MBUS_DT_DTFMT_BPP(SRGGB10_1X10, RAW10, BPP10, 10),
+	FMT_MBUS_DT_DTFMT_BPP(SBGGR10_1X10, RAW10, BPP10, 10),
 	FMT_MBUS_DT_DTFMT_BPP(SGBRG10_1X10, RAW10, BPP10, 10),
 	FMT_MBUS_DT_DTFMT_BPP(SGRBG10_1X10, RAW10, BPP10, 10),
 	FMT_MBUS_DT_DTFMT_BPP(SRGGB10_1X10, RAW10, BPP10, 10),
 	/* 12 bit */
-	FMT_MBUS_DT_DTFMT_BPP(SRGGB12_1X12, RAW12, BPP12, 12),
+	FMT_MBUS_DT_DTFMT_BPP(SBGGR12_1X12, RAW12, BPP12, 12),
 	FMT_MBUS_DT_DTFMT_BPP(SGBRG12_1X12, RAW12, BPP12, 12),
 	FMT_MBUS_DT_DTFMT_BPP(SGRBG12_1X12, RAW12, BPP12, 12),
 	FMT_MBUS_DT_DTFMT_BPP(SRGGB12_1X12, RAW12, BPP12, 12),
 	/* 14 bit */
-	FMT_MBUS_DT_DTFMT_BPP(SRGGB14_1X14, RAW14, BPP14, 14),
+	FMT_MBUS_DT_DTFMT_BPP(SBGGR14_1X14, RAW14, BPP14, 14),
 	FMT_MBUS_DT_DTFMT_BPP(SGBRG14_1X14, RAW14, BPP14, 14),
 	FMT_MBUS_DT_DTFMT_BPP(SGRBG14_1X14, RAW14, BPP14, 14),
 	FMT_MBUS_DT_DTFMT_BPP(SRGGB14_1X14, RAW14, BPP14, 14),
 	/* 16 bit */
-	FMT_MBUS_DT_DTFMT_BPP(SRGGB16_1X16, RAW16, BPP16, 16),
+	FMT_MBUS_DT_DTFMT_BPP(SBGGR16_1X16, RAW16, BPP16, 16),
 	FMT_MBUS_DT_DTFMT_BPP(SGBRG16_1X16, RAW16, BPP16, 16),
 	FMT_MBUS_DT_DTFMT_BPP(SGRBG16_1X16, RAW16, BPP16, 16),
 	FMT_MBUS_DT_DTFMT_BPP(SRGGB16_1X16, RAW16, BPP16, 16),
